lib/workspaces/arborist-cmd.js

```diff
@@ -4,16 +4,22 @@
 
 const BaseCommand = require('../base-command.js')
 class ArboristCmd extends BaseCommand {
+  constructor () {
+    super(...arguments)
+    this.isArboristCmd = true
```

Comment:
Stylistic nit: if this isn't a thing we'd need to change in the life of an object, it should be a getter (so that it'll blow up if we ever try to change it).

So, instead of a constructor, we can just have:

```
  get isArboristCmd () {
    return true
  }
```

Reply:
done